db/init_db.sql

```diff
@@ -0,0 +1,103 @@
+CREATE TABLE IF NOT EXISTS prices (
```

Comment:
Nice. As recommended lecture, and if you are unfamiliar with `flyway`, please research :
- What a db migration tool is? What are the advantages?
- How to include flyway into my java project?